OneCommunityGlobal/HighestGoodNetworkApp
# Prabh removed delete button for Volunteers

# Description
Remove the ability for volunteers to delete tasks from other volunteers
When using a volunteer account and clicking on a task from another volunteer, the delete button will appear and it is possible to delete the task.
…

## Mainly changes explained:
I have added a check in SingleTask.jsx and if user.role is Volunteer the delete button will be not rendered
…

## How to test:
check into the current branch
do `npm install` and `npm run start:local` to run this PR locally
log in as a Volunteer user
go to dashboard→ Tasks and Timelog→ Click on someone's task→ check if delete button appear or not

## Screenshots or videos of changes:
check this video I have explained everything and some issues about testing. Please watch it full


https://user-images.githubusercontent.com/82662768/232076772-afbf3fb8-690c-471e-9174-aec44abbf5b9.mp4



## Note:
Please make sure to watch the video fully.
